core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -2788,6 +2788,17 @@ public static double log(BigDecimal d0, BigDecimal d1) {
     return Math.log(d0.doubleValue()) / Math.log(d1.doubleValue());
   }
 
+  /** SQL {@code LOG2(number)} function applied to double values. */
```

Comment:
in the comment you call the parameter "number" but in the actual method you call it "d0", these should be consistent.

Reply:
I like number